deno/gateway/v10.ts

```diff
@@ -511,7 +512,82 @@ export type GatewayGuildModifyDispatch = DataPayload<
  * https://discord.com/developers/docs/topics/gateway#guild-create
  * https://discord.com/developers/docs/topics/gateway#guild-update
  */
-export type GatewayGuildModifyDispatchData = APIGuild;
+export type GatewayGuildModifyDispatchData = APIGuild & {
```

Comment:
This will also make the GUILD_UPDATE payload have these types. Is that accurate? If not, please fix that

Reply:
Oops, should now be fixed.